dev/com.ibm.ws.kernel.boot/resources/com/ibm/ws/kernel/boot/resources/LauncherMessages.nlsprops

```diff
@@ -772,6 +772,22 @@ error.checkpoint.restore.failed.no.recovery=CWWKE0964E: Restoring the checkpoint
 error.checkpoint.restore.failed.no.recovery.explanation=An error occurred restoring a checkpoint server process. The server did not launch because checkpoint restore recovery is disabled.
 error.checkpoint.restore.failed.no.recovery.useraction=Check the logs to determine why the checkpoint process was not restored. To enable restore failure recovery, set the CRIU_RESTORE_DISABLE_RECOVERY environment variable to true.
 
+warning.serverLockFileNotExist=CWWKE0965W: The server lock file {0} does not exist.
+warning.serverLockFileNotExist.explanation=This may indicate that the server is not currently running or that the lock file has been unexpectedly deleted.
+warning.serverLockFileNotExist.useraction=Check if the server is running using appropriate system monitoring tools. On Windows, you can use the Task Manager. On Unix-like systems, use the 'ps -ef' command.
+
+warning.serverCommandFileNotExist=CWWKE0966W: The server command file {0} does not exist.
+warning.serverCommandFileNotExist.explanation=This suggests that the server command file, which contains essential information for establishing a socket connection with the server, is missing. This could be due to the server not running or the file being unexpectedly deleted.
+warning.serverCommandFileNotExist.useraction=Check if the server is running using appropriate system monitoring tools. On Windows, you can use the Task Manager. On Unix-like systems, use the 'ps -ef' command.  If the server is running, you'll need to restart it, which involves stopping the process.  Use the Task Manager on Windows to stop the process. On Unix-like systems, use the 'kill' command.
```

Comment:
Change to:
```
Check whether the server is running by using appropriate system monitoring 
tools. On Windows, you can use the Task Manager. On UNIX-like systems, 
use the 'ps -ef' command. If the server is running, you must restart it, which 
involves stopping the process. Use the Task Manager on Windows to stop 
the process. On UNIX-like systems, use the 'kill' command.
```
- `if` to `whether`
- `using` to `by using`
- `Unix` to `UNIX` (both places)
- `you'll need to` to `you must`
- Remove extra spaces between sentences.